File: Algorithm/Programmers/Lv1/K번째수.java
import java.util.*;

class Solution {
    public int[] solution(int[] array, int[][] commands) {
                
        int[] answer = new int[commands.length];

        int cnt = 0;
        for(int[] c : commands){
             ArrayList<Integer> list = new ArrayList<>();
             for(int i = c[0]-1; i<c[1]; i++){
                 list.add(array[i]);
             }
            Collections.sort(list);
            answer[cnt++] = list.get(c[2]-1);
        }
        
        return answer;
    }
}

File: Algorithm/Programmers/Sort/K번째수.js
function solution(array, commands) {
    var answer = [];
    var c = [];
    for(let command of commands){
        let i = command[0];
        let j = command[1];
        let k = command[2];
        for(let l = i; l<=j; l++){
            c.push(array[l-1]);
        }
        c.sort( (a, b) => a - b );
        answer.push(c[k-1]);
        c = [];
    }
    return answer;
}
